_includes/disqus_comments.html

```diff
@@ -0,0 +1,25 @@
+{% if page.comments and jekyll.environment == "production" %}
```

Comment:
Does this require every post to have `comments: true?` perhaps make it opt out for posts?